docs/quickstart-aks.md

```diff
@@ -129,7 +97,7 @@ Wait for all the pods in the cluster to enter `Running` state:
   kubectl get pods -n kubeprod
   ```
 
-### Step 4: Registrar setup
+### Step 3: Registrar setup
```

Comment:
To maintain the same style as other steps, I would change this to begin with a verb eg. `Configure DNS records`